kubernetes/ocfweb-web.yaml.erb

```diff
@@ -0,0 +1,72 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ocfweb-web-service
+spec:
+  selector:
+    app: ocfweb-web
+  ports:
+    - port: 80
+      targetPort: 8000
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ocfweb-web-deployment
+  labels:
+    app: ocfweb-web
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ocfweb-web
+  template:
+    metadata:
+      labels:
+        app: ocfweb-web
+    spec:
+      containers:
+        - name: ocfweb-web-deploy
+          image: "docker.ocf.berkeley.edu/ocfweb-web:<%= version %>"
+          resources:
+            limits:
+              memory: 1Gi
+              cpu: 1
+          ports:
+            - containerPort: 8000
+          volumeMounts:
+            - name: ocfweb-secrets
+              mountPath: /etc/ocfweb
+              readOnly: true
+            - name: ocfweb-ocf
+              mountPath: /etc/ocf
+              readOnly: true
+          env:
+            - name: OCFWEB_TESTING
+              value: "0"
+      volumes:
+        - name: ocfweb-secrets
+          hostPath:
+            path: /opt/share/kubernetes/secrets/ocfweb
+            type: Directory
+        - name: ocfweb-ocf
+          hostPath:
+            path: /etc/ocf
+            type: Directory
+      dnsPolicy: ClusterFirst
+      dnsConfig:
+        searches:
+          - "ocf.berkeley.edu"
```

Comment:
@keur why is this necessary?

Reply:
It's when we use OCF hostnames (not fqdn). But it doesn't appear to be necessary. After glancing over the code I can't seem to find an instance where we use one.

Reply:
I'd keep it just in case, it's probably used in various places in ocflib.

Reply:
Do we need `dnsPolicy: ClusterFirst` though?

Reply:
`ClusterFirst` is the default `dnsPolicy` so no we don't need it because it doesn't change anything. For clarity, when using `dnsConfig`, I always include it. I'm not particularly attached if you want to remove it.

Reply:
in particular, logs are sent to "syslog"